### ISSUE TITLE
Use of 'default' as property name causes problems in old IE

### ISSUE THREAD

**Issue by dpvc** [mathjax/MathJax-a11y#202]
The us of `default` as a property in the accessibility-menu extension (which is now included in all the default configurations) causes IE8 to fail when loading the configuration, and that prevents MathJax from running in IE8.  That is true whether any of the accessibility extensions are active or not (since it is in the menu extension itself).

**Comment by dpvc** [comment]
==> Merged.
